index.html

```diff
@@ -0,0 +1,348 @@
+    <nav class="desk-header-left">
+      <a href="" class="desk-porti desk-button"> Portfolio</a>
+      <a href="" class="desk-abt desk-button">About</a>
+      <a href="" class="desk-contact desk-button"> Contact</a>
+      <nav class="desk-nav desk-button">
+        <img src="assets/images/Union.png" alt="M">
+      </nav>
+    </nav>
+  </header>
+  <section class="first-page">
+    <div class="headline">
+      <h1 class="hey">Hey There I'm Solomon</h1>
+      <h2 class="softwaredev"> I am software developer</h2>
+      <p class="headline-p">I can help you build a product , feature or website Look through some of my work
+        and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+    </div>
+
+    <div class="socio-image">
+      <ul class="socila-link">
+        <li class="social"><a href="https://github.com/solog0039"><img src="./assets/socialmedia/github.png"
+              alt="Github"></a></li>
+        <li class="social"><a href="www.linkedin.com/in/solomon-kidanu-62a994232"><img
+              src="./assets/socialmedia/linkidin.png" alt="linkedin"></a></li>
+        <li class="social"><a href="##"><img src="./assets/socialmedia/tump.png" alt="tum"></a></li>
+        <li class="social"><a href="https://twitter.com/Solomon57320119"><img src="./assets/socialmedia/tiwtter.png"
+              alt="tiwtter"></a></li>
+        <li class="social"><a href="##"><img src="./assets/socialmedia/m.png" alt="M"></a></li>
+      </ul>
+      <img src="./assets/images/Group.png" alt="">
+    </div>
+  </section>
+  <section class="desk-first-page">
+    <div class="desk-headline">
+      <h1 class="desk-hey">Hey There I'm Solomon</h1>
+      <h2 class="desk-softwaredev"> I am software developer</h2>
+      <p class="desk-headline-p">I can help you build a product , feature or website Look through some of my work
+        and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+    </div>
+    <ul class="socila-link">
+      <li class="social"><a href="https://github.com/solog0039"><img src="./assets/socialmedia/github.png"
+            alt="Github"></a></li>
+      <li class="social"><a href="www.linkedin.com/in/solomon-kidanu-62a994232"><img
+            src="./assets/socialmedia/linkidin.png" alt="linkedin"></a></li>
+      <li class="social"><a href="##"><img src="./assets/socialmedia/tump.png" alt="tum"></a></li>
+      <li class="social"><a href="https://twitter.com/Solomon57320119"><img src="./assets/socialmedia/tiwtter.png"
+            alt="tiwtter"></a></li>
+      <li class="social"><a href="##"><img src="./assets/socialmedia/m.png" alt="M"></a></li>
+    </ul>
+    <img src="./assets/images/Gourp1-desktop.png" alt="Image in the first page">
+  </section>
+  <section class="cards">
+    <div class="h21-line">
+      <h1 class="h21">My Recent Works</h1>
+      <hr class="line">
```

Comment:
- [ ] As stated by the previous reviewer, please make sure to reduce the space around the `<hr>` , you can do so by setting its width to `width: 55%;` and its margin to `margin-left: 4%;` , please refer to the image below 🔽 for more info.

![header](https://user-images.githubusercontent.com/104319462/221052433-2e89a4c4-8f84-479a-be3b-543b43c89f16.png)